lib/defaults/defaults.go

```diff
@@ -309,8 +309,8 @@ const (
 )
 
 var (
-	// ResyncInterval is how often tunnels are resynced.
-	ResyncInterval = 5 * time.Second
+	// ResyncInterval is how often tunnels are resynced. This lines up with default prometheus scrape interval
+	ResyncInterval = 15 * time.Second
```

Comment:
So this is only used for gathering metrics and nothing more?

If so, the change seems fine but I would rename the variable.

If not, I would want to understand the impacts of changing this value.